data.sql

```diff
@@ -1,5 +1,8 @@
-/* Populate database with sample data. */
+/* Day one queries */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  (1, 'Agumon', '2020-02-03', 0, true, 10.23),
+  (2, 'Gabumon', '2018-11-15', 2, true, 8),
+  (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
+  (4, 'Devimon', '2017-05-12', 5, true, 11);
```

Comment:
please as seen requirements the values required to fill in are float numbers not whole number as insignificant as it may be it plays an important role in having accuracy when dealing with numbers in the database 😄

Reply:
- Well resolved @Victoria129 👍🏽